Assets/Scripts/AnimateWater.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimateWater : MonoBehaviour
```

Comment:
Can you read through this and change some stuff?
https://paper.dropbox.com/doc/General-Project-Conventions--ApG2LipOwhFvfjJLQeJpIQSlAg-Z8kbXCDr3XqW11DkCtUHe

Reply:
will do. i also wanna wait with the PR until we have gamescene in master